.github/workflow/ci_pipeline.yml

```
@@ -0,0 +1,45 @@
+name: CI Pipeline
+on: 
+  push:
+    branches:
+     - 'feature/**'
+jobs:
+  ktlint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)